translations/ui-agreements/en.json

```diff
@@ -35,6 +35,7 @@
   "agreements.agreementStatus": "Status",
   "agreements.isPerpetual": "Is perpetual",
   "agreements.internalContacts": "Internal contacts",
+  "agreements.internalContactsRole": "Internal contacts role",
```

Comment:
I think `Internal contact role` (singular) would be better and would also match the verbiage used for `Organization role`